sympy/polys/polytools.py

```diff
@@ -4408,7 +4408,7 @@ def degree(f, gen=0):
     degree_list
     """
 
-    if f.is_Poly:
+    if isinstance(f, Poly):
```

Comment:
This change is probably not necessary. I think that the `.is_Poly` attribute was introduced for exactly the same purpose. (See `git show 39f4887c2`.)